pkg/users/accounts-list.js

```diff
@@ -129,7 +105,17 @@ const getGroupRow = (group, accounts) => {
     const columns = [
         {
             sortKey: group.name,
-            title: <Flex alignItems={{ default: 'alignItemsCenter' }}><div className={"dot " + groupColorClass} /><FlexItem>{group.name}</FlexItem></Flex>,
+            title: (
+                <Flex alignItems={{ default: 'alignItemsCenter' }}>
+                    <div className={"dot " + groupColorClass} />
+                    <FlexItem>
+                        <a onClick={ev => { cockpit.jump("users#/group/" + group.name); ev.preventDefault() }}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18136-20230302-130300-75913373-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)